pd/src/state/reader.rs

```diff
@@ -112,6 +112,19 @@ impl Reader {
         Ok(note_commitment_tree)
     }
 
+    /// Returns the number of revealed nullifiers (for updating metrics dashboard).
+    pub async fn nullifier_count(&self) -> Result<u64> {
+        let mut conn = self.pool.acquire().await?;
+
+        // Special column name "count!" is used to return i64 instead of
+        // Option<i64> due to https://github.com/launchbadge/sqlx/issues/864
+        let nullifiers = query!(r#"SELECT COUNT(*) as "count!" FROM nullifiers"#)
+            .fetch_one(&mut conn)
+            .await?;
+
+        Ok(nullifiers.count.try_into().unwrap())
+    }
+
```

Comment:
Since we'd never want the nullifier count for any reason other than metrics/statistics, I think it might be better if the method were a "fetch chain stats / metrics / etc" that returned a bundle of data, so that as we want to add more metrics, we can add fields to the returned data and still only have to do one query.